.github/workflows/bump-sentry-in-getsentry.yml

```
@@ -0,0 +1,54 @@
+name: bump sentry in getsentry
+
+# Serializes the execution of this workflow.
+concurrency:
+  group: ${{ github.workflow }}
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  bump-sentry:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28  # v3
+        with:
+          # For bin/bump-sentry, sentry needs to be at ../sentry relative to getsentry.
+          path: sentry
+
+      - name: checkout getsentry
+        uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28  # v3
+        with:
+          repository: 'getsentry/getsentry'
+          path: getsentry
+          # This PAT (Personal Access Token) belongs to getsentry-bot,
```

```suggestion
          # This Personal Access Token belongs to getsentry-bot,
```

nit: I would use the acronym if you're gonna say it again later in this file, but we don't